README.md

```diff
@@ -91,7 +91,13 @@ $ brew install lima
 - On ARM hosts, QEMU has to be patched for enabling `--accel=hvf` support. See https://gist.github.com/nrjdalal/e70249bb5d2e9d844cc203fd11f74c55
 
 - Download the binary archive of Lima from https://github.com/lima-vm/lima/releases ,
-and extract it under `/usr/local` (or somewhere else).
+and extract it under `/usr/local` (or somewhere else). For instance, to extract it under `$HOME`:
```

Comment:
The example should be `/usr/local`, not `$HOME`

Reply:
It is a bit of a mess on ARM, as homebrew installs packages compiled for ARM into `/opt/homebrew` and packages compiled for Intel (using Rosetta) into `/usr/local`. Not sure if we need to acknowledge that, or if we should always install into `/usr/local`?

Reply:
Dear @AkihiroSuda and @jandubois for the reason in macOS, `System Integrity Protection` was introduced years ago, thus I didn't assume all users could disable `System Integrity Protection` to place files under `/usr/local`.

What do you think please, which example path should it be in this example line?

Reply:
> I didn't assume all users could disable `System Integrity Protection` to place files under `/usr/local`.

You do not need to disable SIP to create directories in `/usr/local`; you just need to be `root`. Otherwise `brew` would not work, which create various directories under `/usr/local` and changes ownership to the local user (so further installs can be done without `sudo`).

What you **cannot** do without disabling SIP is creating additional directories directly under `/usr` (tested on Monterey Public Beta):

```console
jan@Jans-Mac ~ % sudo mkdir -p /usr/local/foo
Password:
jan@Jans-Mac ~ % sudo mkdir -p /usr/foo      
mkdir: /usr/foo: Operation not permitted
```

Reply:
> You do not need to disable SIP to create directories in /usr/local; you just need to be root. Otherwise brew would not work, which create various directories under /usr/local and changes ownership to the local user (so further installs can be done without sudo).

Got it, thank you Jan, and now I come to learn how brew works now! will update patch to use `/usr/local` with root

Reply:
I think previously homebrew used some hack to change the owner of `/usr/local`

There was some tension between that chmod and using `/opt/local` (like MacPorts)

Reply:
> I think previously homebrew used some hack to change the owner of /usr/local
>
> There was some tension between that chmod and using /opt/local (like MacPorts)

Aha, thank you for the sharing, I didn't know this history and as brew was installed, thus was not aware of the ownership change on `/usr/local`.